### PR TITLE
docs: add hints to interact with system dialogs

### PR DESCRIPTION
**Pull request by KazuCocoa**
Maybe it would be nice to address the dialog thing in troubleshooting

### CODE REVIEW

File: docs/guides/troubleshooting.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ title: Troubleshooting
   [this issue](https://github.com/facebook/WebDriverAgent/issues/507) for more details.
 * `shake` is implemented via AppleScript and works only on Simulator due to lack of support from Apple
 
+## Interact with dialogs managed by `com.apple.springboard`
+
+System dialogs, such as permission dialogs, might not be interactable directly when the active application is not `com.apple.springboard`
+because of the current active application. For example, when a session starts with `appium:app` or `appium:bundleId` capability with
```
Comment (mykola-mokhnach) [inline docs/guides/troubleshooting.md:21]: > because of the current active application

might be removed

Comment (mykola-mokhnach) [reply docs/guides/troubleshooting.md:21]: For example, when a session starts with  ...

This sentence is not really needed, the previous one already describes the same

File: docs/guides/troubleshooting.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ title: Troubleshooting
...
+non-springboard. Despite a similar look, dialogs belonging to the `appium:app` or `appium:bundleId` do not require such adjustment.
```
Comment (mykola-mokhnach) [inline docs/guides/troubleshooting.md:22]: dialogs belonging to the active session application (e.g. initially passed as `appium:app` or `appium:bundleId` capability value)

File: docs/guides/troubleshooting.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ title: Troubleshooting
...
+
+XCUITest driver offers a few methods to handle them.
+
+- [`mobile: alert`](../reference/execute-methods.md#mobile-alert)
+- Enable `appium:autoAcceptAlerts`/`appium:autoDismissAlerts`, or interact with alerts via [User Prompts](https://www.w3.org/TR/webdriver1/#user-prompts) in WebDriver endpoints
+    - e.g. `driver.switch_to.alert.accept`
```
Comment (mykola-mokhnach) [inline docs/guides/troubleshooting.md:28]: Please mention the example is for the Ruby client

File: docs/guides/troubleshooting.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ title: Troubleshooting
...
+    - It might need to coordinate element selection with `acceptAlertButtonSelector`/`dismissAlertButtonSelector` in [Settings](../reference/settings.md)
```
Comment (mykola-mokhnach) [inline docs/guides/troubleshooting.md:29]: it might be necessary ... via .... settings

**Comment by KazuCocoa** [comment]
updated

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**